metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java

```diff
@@ -409,19 +410,20 @@ public void mergeEntities(RDF rdf, List<EnrichmentBaseWrapper> enrichmentBaseWra
    * @param proxyLinkTypes Lookup of the link types to create in the europeana proxy. The keys are
    * the about values of the entities to add.
    */
-  public void mergeEntities(RDF rdf, List<EnrichmentBase> contextualEntities,
+  public void mergeEntities(RDF rdf, List<List<EnrichmentBase>> contextualEntities,
```

Comment:
Probably nicer to make this a list of entries, not a nested list, and to do a flatMap in the calling method. Then I think you can revert the changes in this method.

Reply:
I also think it is a better idea. However, if I call the method like this `mergeEntities(rdf, enrichmentBaseList, null)` it becomes ambiguous